site/_apps/inferno-hn.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 24
+title: 'Inferno HN'
+github-title: 'aretheregods/hackerinferno'
+libraries:
+    - name: 'Inferno'
+scaffolding: 'Create-Inferno-App'
+module-bundling: 'Webpack'
+service-worker: 'Application Shell + data caching with sw-precache'
+perfomance-patterns: 'PRPL, code-splitting for granular loading'
+server-side-rendering: 'None'
+api: 'HNPWA-API (unofficial)'
+hosting: 'Firebase functions over HTTP/2 with Server Push + edge-caching'
+other-details: 'Route-specific pushing'
+authors:
+    - name: 'Naythan'
+lighthouse: 91/100
+interactive-em: 2.6s
```

Comment:
JavaScript is settling on the main thread in under 2.7 seconds here so this looks fine: https://www.webpagetest.org/video/compare.php?tests=171014_9P_6a3d5a195dc7b31e50e4fa94ef908639-r%3A1-c%3A0&thumbSize=200&ival=500&end=full